### ISSUE TITLE
Corrections to punctuation in strings

### ISSUE THREAD

**Issue by terracoda** [phetsims/balloons-and-static-electricity#335]
Sweater Rubbing Alert, found one minor issue.
- [x] Extra periods in Sweater rubbing alert 2-3 (last charge picked up) in Show charge differences view:
  - "Sweater has positive net charge, showing all positive charges..Yellow balloon has negative net charge, showing many negative charges.."
  
Found the following minor punctuation issues while reading through an changed PDOM with cursor keys:
- [x] Balloon description is missing a period at end of location phrase.
- [x] Wall description has two periods at the end of the induced charge phrase.
- [x] Sweater description is missing a period at end of charge description phrase.
- [x] Sim Resources and Tools heading has a period, but that should be removed.

**Comment by jessegreenberg** [comment]
Fixed in the above commits, closing.
